README.md

```diff
@@ -26,6 +26,7 @@ Variety is available in the distro repositories of:
 - [Fedora](https://www.rpmfind.net/linux/rpm2html/search.php?query=variety)
 - [OpenSUSE](https://software.opensuse.org/package/variety?search_term=variety)
 - [Ubuntu 16.04+](https://packages.ubuntu.com/search?keywords=variety)
+- [NixOS](https://search.nixos.org/packages?channel=23.05&type=packages&query=variety)
```

Comment:
Thanks for the PR. Let's not hardcode a channel version, this goes obsolete too quickly. And let's make sure the "variety" package is shown expanded.
```suggestion
- [NixOS](https://search.nixos.org/packages?show=variety&from=0&size=50&sort=relevance&type=packages&query=variety)
```